horizontalintegration/xmc-hztl-foundation-2024-q1
# Bug NOJIRA-0000 Accordion UX Enhancements

## What does this PR do and why?

This PR includes some code clean-up and adds animation to the Accordion Item component.

## Screenshots or screen recordings

N/A

## How to set up and validate locally

1. Check out the branch _bug/NOJIRA-0000-accordion-u_.
2. From a command prompt, navigate to _src/hztl-foundation_ and execute `npm run storybook`.
3. Once Storybook loads, navigate to _Components/Authorable/Shared/Lists/Accordion Item_ and confirm that the component operations normally.

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
